src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -179,13 +188,28 @@ function FloatingActionButtonAndPopover(props) {
                         text: translate('iou.sendMoney'),
                         onSelected: () => interceptAnonymousUser(() => IOU.startMoneyRequest(CONST.IOU.TYPE.SEND)),
                     },
-                    ...[
-                        {
-                            icon: Expensicons.Task,
-                            text: translate('newTaskPage.assignTask'),
-                            onSelected: () => interceptAnonymousUser(() => Task.clearOutTaskInfoAndNavigate()),
-                        },
-                    ],
+                    ...(canUseTrackExpense
+                        ? [
+                              {
+                                  icon: Expensicons.TrackExpense,
+                                  text: 'Track Expense',
+                                  onSelected: () =>
+                                      interceptAnonymousUser(() =>
+                                          IOU.startMoneyRequest_temporaryForRefactor(
+                                              CONST.IOU.TYPE.TRACK_EXPENSE,
+                                              // When starting to create a track expense from the global FAB, we need to retrieve selfDM reportID.
+                                              // If it doesn't exist, we generate a random optimistic reportID and use it for all of the routes in the creation flow.
+                                              props.account.selfDMReportID || ReportUtils.generateReportID(),
```

Comment:
Actually, can we instead add a method in ReportUtils that will get the selfDM report? We can't add the onyx key until all users are given the self-DM, and even then if we can get this reportID on the front-end from the reports in onyx, that is preferred